src/Sylius/Behat/Page/Shop/Checkout/AddressPage.php

```diff
@@ -266,10 +266,12 @@ protected function getDefinedElements(): array
             'customer_email' => '[data-test-login-email]',
             'different_billing_address' => '[data-test-different-billing-address]',
             'different_billing_address_label' => '[data-test-different-billing-address-label]',
+            'different_shipping_address' => '[data-test-different-shipping-address]',
+            'different_shipping_address_label' => '[data-test-different-shipping-address-label]',
             'login_button' => '[data-test-login-button]',
             'login_password' => '[data-test-password-input]',
             'next_step' => '[data-test-next-step]',
-            'shipping_address_book' => '[data-test-address-book]',
+            'shipping_address_book' => '[data-test-shipping-address] .ui.dropdown',
```

Comment:
Can you replace `.ui.dropdown` with some data attribute?

Reply:
it's probably should looks like:
```suggestion
            'shipping_address_book' => '[data-test-shipping-address] [data-test-address-book]',
```